src/twisted/application/internet.py

```diff
@@ -1149,9 +1197,9 @@ def stopService(self):
         return self._machine.stop()
 
 
+
 __all__ = (['TimerService', 'CooperatorService', 'MulticastServer',
             'StreamServerEndpointService', 'UDPServer',
-            'ClientService'] +
-           [tran + side
-            for tran in 'TCP UNIX SSL UNIXDatagram'.split()
-            for side in 'Server Client'.split()])
+            'ClientService', 'TCPServer', 'TCPClient',
+            'UNIXServer', 'UNIXClient', 'SSLServer', 'SSLClient',
+            'UNIXDatagramServer', 'UNIXDatagramClient'])
```

Comment:
The outer parentheses no longer serve a purpose now that everything is in a single list.

Reply:
Fixed